src/jobs/lint-app.yml

```diff
@@ -1,9 +1,90 @@
 description: >
   This job will lint the application based on what language the application is written in.
 
-executor: node/default
+parameters:
+  # Not Required Parameters
+  docker-image:
+    type: string
+    default: "cimg/base:stable"
+    description: |
+      The docker image that will be used as an executor
+      if isNode is true, node/default is recommended
+      if isGolang is true, golangci/golangci-lint:latest-alpine is recommended
+      if isPython is true, cimg/python:3.10 is recommended
+  isNode:
```

Comment:
For parameters like isNode, isGolang, & isPython, do we have to manually set the default value or can we set this dynamically through a CLI command?